### PR TITLE
userSchema

### PR DESCRIPTION
**Pull request by Liettualainen**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DB_HOST=mongodb+srv://Mykhailo:vBADnnnXh7t9MA2u@cluster0.zkkdc5c.mongodb.net/db-contacts?retryWrites=true&w=majority
```
Comment (ceREAL133) [inline .env]: this is sensitive information. File .env must be hidden, add it to gitignore

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (Liettualainen) [reply .env]: have already corrected

Comment (ceREAL133) [reply .env]: why am i stiil able to see it here?

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (Liettualainen) [reply .env]: I supposed that after I decomment env in gitignor, file env would be deleted from git. Now I  deleted it, than  I created the same one but mentioned it in gitignor.

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**